SmartTutor/ContentModel/ContentService.cs

```diff
@@ -63,13 +66,16 @@ public NodeProgress GetNodeContent(int knowledgeNodeId, int? traineeId)
 
         private NodeProgress CreateNodeForTrainee(int knowledgeNodeId, int? traineeId)
```

Comment:
We can change the signature to have a mandatory traineeId, and add an explicit (int) cast in line 51.